src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,31 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public int hashCode() {
+        return 31 * getLength() * getWidth()
```

Comment:
what will happen if width or length ==0?
try do smth like this
https://www.baeldung.com/java-hashcode

Reply:
what will happen if length or width is 0?

can we use a hashcode of width, length and color and other algorithm of calculation?